pkg/admissioncontroller/apis/config/v1alpha1/defaults_test.go

```diff
@@ -24,69 +24,121 @@ import (
 )
 
 var _ = Describe("Defaults", func() {
-	Describe("AdmissionControllerConfiguration", func() {
-		var obj *AdmissionControllerConfiguration
+	var obj *AdmissionControllerConfiguration
 
-		Context("Empty configuration", func() {
-			BeforeEach(func() {
-				obj = &AdmissionControllerConfiguration{}
-			})
+	BeforeEach(func() {
+		obj = &AdmissionControllerConfiguration{}
+	})
 
-			It("should correctly default the admission controller configuration", func() {
-				SetObjectDefaults_AdmissionControllerConfiguration(obj)
+	Describe("AdmissionControllerConfiguration defaulting", func() {
+		It("should correctly default the AdmissionControllerConfiguration", func() {
+			SetObjectDefaults_AdmissionControllerConfiguration(obj)
+			// ContentType fields will be defaulted by client constructors / controller-runtime based on whether a
+			// given APIGroup supports protobuf or not. defaults must not touch these, otherwise the integelligent
+			// logic will be overwritten
+			Expect(obj.GardenClientConnection.ContentType).To(BeEmpty())
+			Expect(obj.GardenClientConnection.AcceptContentTypes).To(BeEmpty())
+			Expect(obj.GardenClientConnection).To(Equal(componentbaseconfigv1alpha1.ClientConnectionConfiguration{
+				QPS:   50.0,
+				Burst: 100,
+			}))
```

Comment:
This should be moved under separate `Describe("GardenClientConnection defaulting", func() {})` block